src/components/ProductGridContainer.tsx

```diff
@@ -2,15 +2,18 @@ import { useState } from "react";
 import ProductGrid from "./ProductSearch";
 import { ProductsType } from "@/service/types";
 
+const url = "https://dummyjson.com/products";
```

Comment:
I believe you could put this url as a constant in `service/constants.ts` and avoid pass it as a prop since it will never change

```js
const base_url = "https://dummyjson.com"
const products = "/products"
```